### PR TITLE
Added tags parameter for smartcontent website URL

### PR DESCRIPTION
**Pull request by wachterjohannes**
__tasks:__

- [x] test coverage
- [ ] gather feedback for my changes
- [x] submit changes to the documentation

__Related PR:__

* https://github.com/sulu-io/sulu-standard/pull/500
* https://github.com/sulu-io/sulu-docs/pull/64

__informations:__

| q                | a
| ---------------- | ---
| Fixed tickets    | none
| BC breaks        | none
| Documentation PR | none

### CODE REVIEW

**Comment by wachterjohannes** [comment]
following missing feature: the developer should be able to define which operator should be used for the conjunction of the tags from the website URL (over a param). This feature have to be activated over a param.

The tags should be combined following:

* Configured tags in the overlay with the configured operator
* AND
* Tags from the URL with the operator from the param

File: tests/Sulu/Component/SmartContent/ContentTypeTest.php [hunk 1]
```diff
@@ -26,7 +27,8 @@
 /**
  * @group unit
  */
-class ContentTypeTest extends \PHPUnit_Framework_TestCase
+class
+ContentTypeTest extends \PHPUnit_Framework_TestCase
```
Comment (wachterjohannes) [inline tests/Sulu/Component/SmartContent/ContentTypeTest.php]: style (:

File: src/Sulu/Component/SmartContent/ContentType.php [hunk 2]
```diff
@@ -216,11 +226,25 @@ public function getContentData(PropertyInterface $property)
...
+            $filters['tags'] = [];
+        }
+
+        // extends selected filter with requested tags
+        $filters['websiteTags'] = $this->tagRequestHandler->getTags($params['tags_parameter']->getValue());
+        $filters['websiteTagOperator'] = $params['website_tag_operator']->getValue();
+
         // resolve tags to id
-        if (!empty($filters['tags'])) {
+        if (!empty($filters['tags']) && count($filters['tags']) > 0) {
```
Comment (danrot) [inline src/Sulu/Component/SmartContent/ContentType.php]: Is this addition really necessary? If the count of an array is 0, empty should also return false.

File: src/Sulu/Component/SmartContent/ContentType.php [hunk 3]
```diff
@@ -216,11 +226,25 @@ public function getContentData(PropertyInterface $property)
...
             $filters['tags'] = $this->tagManager->resolveTagNames($filters['tags']);
         }
 
+        // resolve website tags to id
+        if (!empty($filters['websiteTags']) && count($filters['websiteTags']) > 0) {
```
Comment (danrot) [inline src/Sulu/Component/SmartContent/ContentType.php]: Same as above.
